samples/subsys/shell/shell_module/src/main.c

```diff
@@ -447,4 +449,11 @@ void main(void)
 		k_sleep(K_MSEC(100));
 	}
 #endif
+
+	// prevent system from going into deep sleep by doing some dummy work
+	while(1)
```

Comment:
With this loop we would say to the user that shell requires system to wake up every 1 ms which is not true. It's the specific power management handling of that board and in that board it should be solved. Maybe there should be a Kconfig option which tells power management to not turn off the UART and it would be enabled by default when shell is on.

Reply:
Better to disable CONFIG_PM at board level KConfig 
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig#L20

That need to ensure CONFIG_PM is enabled correctly for MEC152x samples board code 
samples/boards/mec15xxevb_assy6853/power_management/prj.conf 
ping @jvasanth1

Reply:
agree with @albertofloyd, we can remove CONFIG_PM from board level KConfig

Reply:
@ambranaam Remove the while and just try by removing this line
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig#L21